Tasks/Common/azure-blobstorage-artifactProvider/blobservice.ts

```diff
@@ -14,10 +14,14 @@ export class BlobService {
         this._storageAccessKey = storageAccessKey;
     }
 
-    public async uploadBlobs(source: string, container: string, prefixFolderPath?: string, host?: string): Promise<string[]> {
+    public async uploadBlobs(source: string, container: string, prefixFolderPath?: string, host?: string, itemPattern?: string): Promise<string[]> {
         var fileProvider = new artifactProviders.FilesystemProvider(source);
         var azureProvider = new azureBlobProvider.AzureBlobProvider(this._storageAccountName, container, this._storageAccessKey, prefixFolderPath, host);
         var processor = new artifactProcessor.ArtifactEngine();
+        var processorOptions =  new artifactProcessor.ArtifactEngineOptions();
```

Comment:
Should we have a GetDefaultOptions() function in ILD ArtifactEngineOptions which returns default value. ArtifactEngine should use same default values. This will safeguard against diverging default options between us and ILD

Reply:
The constructor initializes all with default value, hence diversion is not possible as of today.